lib/ansible/plugins/lookup/env.py

```diff
@@ -31,21 +31,21 @@
     msg: "'{{ lookup('ansible.builtin.env', 'HOME') }}' is the HOME environment variable."
 
 - name: Before 2.13, how to set default value if the variable is not defined.
-        This cannot distinguish between USR undefined and USR=''.
+        This cannot distinguish between USER undefined and USER=''.
   ansible.builtin.debug:
-    msg: "{{ lookup('ansible.builtin.env', 'USR')|default('nobody', True) }} is the user."
+    msg: "{{ lookup('ansible.builtin.env', 'USER')|default('nobody', True) }} is the user."
```

Comment:
`USR` is an environment variable which the user does not define to show `|default` example. I agree this is the wrong choice for a variable name. 
Could you please replace this variable name with something `VARIABLE_NOT_DEFINED_IN_THE_ENV` or something along those lines? This will show how the user can use default values if the environment variable is not defined.

Reply:
Personally I think using USER would be a better choice, as this is an example that demonstrates how the lookup plugin works, but not the default filter. By changing this to an undefined variable will bring more confusion. My two cents.

Reply:
the whole point was to use variable that was mispelled/undefined, so `USER` would be a bad choice

Reply:
Thanks for your explanation, you are right, so the undefined one would be a better choice under this situation.